packages/plugins/experimental/plugin-wnfs/src/blockstore.ts

```diff
@@ -0,0 +1,82 @@
+//
+// Copyright 2024 DXOS.org
+//
+
+import { BaseBlockstore } from 'blockstore-core';
+import { IDBBlockstore } from 'blockstore-idb';
+import type { CID } from 'multiformats';
+
+import { storeName } from './common';
+
+export const create = () => {
+  return new MixedBlockstore();
+};
+
+/**
+ * A blockstore that communicates with the DXOS blob service,
+ * and which uses an indexedDB blockstore as a client cache.
+ */
+class MixedBlockstore extends BaseBlockstore {
+  readonly #apiHost: string;
+  readonly #idbStore: IDBBlockstore;
+
+  constructor() {
+    super();
+
+    this.#apiHost = 'http://localhost:8787';
```

Comment:
@wittjosiah This is supposed to target the Blob service, is there a production instance yet?

Reply:
@richburdon Any thoughts?

Reply:
obtain from the config object (useConfig).
add `runtime.services.blobstore.endpoint`